src/styles/App.css

```diff
@@ -25,4 +25,10 @@
 @keyframes App-logo-spin {
   from { transform: rotate(0deg); }
   to { transform: rotate(360deg); }
+}
+
+/* To make Pop-up more readable */
```

Comment:
+1 - but does the popup from mapboxgl come with different styles? (it's ok if it doesn't)

Reply:
Yep normally looks like this:

<img width="134" alt="ssmap" src="https://user-images.githubusercontent.com/20118952/33339589-82b0b5c6-d42e-11e7-9b26-0fefcf11b763.png">